libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -1297,11 +1301,9 @@ mergeSolverStage (SolverStage x ps spec) plan =
               ( NE.fromList
                   ( [ "Solving var " ++ show x ++ " fails."
                     , "Merging the Specs"
-                    , "   " ++ show spec
-                    , "   " ++ show spec'
-                    , "Predicates"
+                    , "   1. " ++ show spec
+                    , "   2. " ++ show spec'
                     ]
-                      ++ (map show (ps ++ ps'))
```

Comment:
Too much noise for  value to show these?

Reply:
Yes, and they are already present elsewhere in those error messages.